### ISSUE TITLE
Incorrect svg file is generated

### ISSUE THREAD

**Issue by igorwojda** [misakuo/svgtoandroid#21]
Missing path tag

Attach (No way to attach svg here so I had to zip it)
[ic_star_big_filled.zip](https://github.com/misakuo/svgtoandroid/files/847844/ic_star_big_filled.zip)

**Comment by igorwojda** [comment]
It looks like Android studio does not accept all the attributes inside svg.
You should display error message instead of creating file

**Comment by misakuo** [comment]
You are right, the VectorDrawable only implements a subset of SVG, maybe we can give users some friendly tips, even we can do more compatibility with the SVG format. I will try to achieve it.    
By the way, now there are actually some log that will be output to `Event Log` on IDE.

**Comment by igorwojda** [comment]
To be honest I havent noticed the log and feeld a little bit confusing, when 'empty' VectorDrawable was created - there is no point of creating it.

Also I have noticed that this website does much better job then Android Studio buildin tools
http://inloop.github.io/svg2android/

(just sending info, it seems to be open source project, mabye it will be helpfull)

**Comment by misakuo** [comment]
Yes, we should give the user more obvious tips when the exception occurs, like an alert dialog or popup window ......      

Welcome to open issue here when converted wrongly, it will help me improve the compatibility of plugin.
